bash_completion

```diff
@@ -98,16 +98,18 @@ complete -b builtin
 
 # Check if we're running on the given userland
 # @param $1 userland to check for
-_userland()
+_comp_get_userland()
```

Comment:
```suggestion
_comp_userland()
```

I feel the `get_` here in the name is redundant, and inconsistent with what we have in the codebase already. For example, none of the `_comp_*` in `completions/` have it, even though they are "getters". And this actually is not even a getter -- `is_` would be more appropriate, but I think without either is fine.

Reply:
Thank you! Actually, these naming are something that I wanted to discuss when the PR is ready for review. I will adopt these suggestions.